net/gce.sh

```diff
@@ -114,6 +116,9 @@ while getopts "h?p:Pn:c:z:gG:a:d:" opt; do
   z)
     cloud_SetZone "$OPTARG"
     ;;
+  r)
```

Comment:
would have assumed the same sense as fullnode.rs's, i.e. a config parameter that would eventually be remove-able because it'd be the default